cloud/pkg/controller/manager/location.go

```diff
@@ -139,9 +141,27 @@ func (lc *LocationCache) DeleteSecret(namespace, name string) {
 
 // DeleteNode from cache
 func (lc *LocationCache) DeleteNode(name string) {
-	for i, v := range lc.edgeNodes {
+	for i, v := range lc.EdgeNodes {
```

Comment:
lc.EdgeNodes.Delete(name)

Reply:
used delete(map,key) as i have used map instead of sync.Map